tecken/upload/views.py

```diff
@@ -315,7 +300,7 @@ def upload_archive(request, upload_workspace):
             executor.submit(
                 upload_file_upload,
                 bucket_info,
-                key_name,
+                member.name,
```

Comment:
`member.name` is the key without the prefix, as it is used by the backend interface.

Reply:
I know it's confusing, and I actually confused myself. My intention was to change the code in a way that we still store the prefix in the database, but use the key without the prefix for the backend code, as a compromise to avoid the big data migration. However, this isn't what the code is doing. New entries won't have the prefix included in the db.

I think the only reasonable way to avoid the data migration is to exclude the prefix from the database for new file uploads, and fix it upon loading for existing ones. I'll go ahead and do that.